tonyear/core.py

```diff
@@ -90,7 +90,7 @@ def calculate_tonyears(
     Parameters
```

Comment:
I think it's worthwhile adding a note of caution about the `discount_rate` parameter and maybe pointing the reader to the docs. Definitely don't have to get into it too much but maybe something like:

"Extreme caution should be used when discounting IRF curves. See documentation."